source/navitiacommon/navitiacommon/default_values.py

```diff
@@ -157,12 +157,12 @@ def get_value_or_default(attr, instance, instance_name):
         value = getattr(sys.modules[__name__], attr)
         if not instance:
             logger = logging.getLogger(__name__)
-            logger.warning(
-                'instance %s not found in db, we use the default value (%s) for the param %s',
-                instance_name,
-                value,
-                attr,
-            )
+            # logger.warning(
+            #     'instance %s not found in db, we use the default value (%s) for the param %s',
+            #     instance_name,
+            #     value,
+            #     attr,
+            # )
```

Comment:
Why not simply delete it?

Reply:
I put it back. I think it was annoying me when looking at logs when developing.